### PR TITLE
Runtsafe order bug

### PR DESCRIPTION
**Pull request by jnwatson**
`[test:str=foo :hehe=bar] $tvar=$lib.text('') $tvar.add(1) $tvar.add(:hehe) $lib.print($tvar.str()) `
would evaluate adding :hehe and 1 in the wrong order.

Also fix a bug in error printing in cmdr on a long line with the syntax error near the beginning of the line.

### CODE REVIEW

File: synapse/lib/ast.py [hunk 1]
```diff
@@ -521,19 +521,17 @@ class VarEvalOper(Oper):
     $foo.bar("baz")
     '''
     async def run(self, runt, genr):
+        id = s_common.guid(str(self))[:4]
```
Comment (invisig0th) [inline synapse/lib/ast.py]: this seems like vestigial debug code?  also, id is a python builtin and should be avoided :D

Comment (vEpiphyte) [inline synapse/lib/ast.py]: reusing a builtin function here

Comment (vEpiphyte) [reply synapse/lib/ast.py]: and this is unused?

File: synapse/tests/test_cortex.py [hunk 2]
```diff
@@ -2768,6 +2768,12 @@ class Bawk:
             nodes = await core.nodes(q)
             self.len(1, nodes)
 
+    async def test_storm_order(self):
+        q = '''[test:str=foo :hehe=bar] $tvar=$lib.text('') $tvar.add(1) $tvar.add(:hehe) $lib.print($tvar.str()) '''
```
Comment (invisig0th) [inline synapse/tests/test_cortex.py]: $lib.text() doesn't need the '' arg

File: synapse/lib/ast.py (hunk 1 above)
Comment (jnwatson) [reply synapse/lib/ast.py]: yes debug code

**Pull request merged**